test/Raven/Tests/ClientTest.php

```diff
@@ -1035,6 +1035,20 @@ public function testCaptureMessageWithUserContext()
         ), $event['user']);
     }
 
+    /**
+     * @covers Raven_Client::capture
+     */
+    public function testRuntimeContext()
+    {
+        $client = new Dummy_Raven_Client();
+
+        $client->captureMessage('test');
+        $events = $client->getSentEvents();
+        $this->assertEquals(PHP_VERSION, $events[0]['contexts']['runtime']['version']);
+        $event = array_pop($events);
+        $this->assertEquals('php', $event['contexts']['runtime']['name']);
```

Comment:
Maybe also add an test what happens if the context was set by an user (for the runtime) it correctly merges with our runtime context.